infra/dev/scripts/setup-postgres-linux.sh

```diff
@@ -53,8 +53,10 @@ TARGETARCH=$(dpkg --print-architecture)
 
 # Install PostgresSQL
 echo_header $GREEN "Step [1/4]: Installing PostgreSQL..."
+sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+wget -qO- https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo tee /etc/apt/trusted.gpg.d/pgdg.asc &>/dev/null
 sudo apt update -y || handle_error "Failed to update package list."
-sudo apt install -y postgresql-$PG_MAIN_VERSION postgresql-contrib || handle_error "Failed to install PostgreSQL."su
+RUNLEVEL=1 sudo apt install -y postgresql-$PG_MAIN_VERSION postgresql-contrib || handle_error "Failed to install PostgreSQL."su
```

Comment:
and this one?

Reply:
It appears to me that this might have no effect on the running of this script. We're going to run this again without it to confirm.

Reply:
Reverted this change